go.mod

```diff
@@ -356,3 +356,5 @@ require (
 	golang.org/x/time v0.8.0
 	gopkg.in/yaml.v3 v3.0.1
 )
+
+replace nhooyr.io/websocket => github.com/coder/websocket v1.8.7
```

Comment:
hmm, I am not getting any error related to this when I run `go mod tidy`?

Reply:
I am on latest `main` and everything seems fine.

Reply:
@pxp928 
Strangely, I'm still getting this error in my linux machine even though it works fine on mac.
I'll mark this PR as draft for now.